src/mongroup.cpp

```diff
@@ -119,10 +115,10 @@ MonsterGroupResult MonsterGroupManager::GetResultFromGroup(
             if( ( elem ) == "SUMMER" || ( elem ) == "WINTER" || ( elem ) == "SPRING" ||
```

Comment:
While you are in this file can you replace `( elem )` with simply `elem`?

Reply:
Done, modified throughout the condition checking loop.